lib/elixir_search_extractor_web/helpers/ecto_error_beautifier_helper.ex

```diff
@@ -0,0 +1,9 @@
+defmodule ElixirSearchExtractorWeb.EctoErrorBeautifierHelper do
```

Comment:
How about renaming this to make it more concise?
```suggestion
defmodule ElixirSearchExtractorWeb.ErrorHelper do
```

Reply:
@hanam1ni I wanted to keep the `Ecto` prefix as it is directly related to ecto changeset. Is that okay? Fixed reviews in [a07d143](https://github.com/rafayet-monon/elixir_search_extractor/pull/53/commits/a07d143f70f58a2f0dde14d64b3aa98a945af02f)